libraries/postgres/test/database_test.js

```diff
@@ -104,4 +104,35 @@ dbSuite(path.basename(__filename), function() {
       await db.close();
     }
   });
+
+  test('procedure methods does not have capital letters', async function () {
+    const schema = Schema.fromSerializable({
+      versions: [{
+        version: 1,
+        migrationScript: `begin
+            create table testing (a integer, b integer);
+          end`,
+        methods: {
+          testData: {
+            mode: 'write',
+            args: '',
+            returns: 'void',
+            body: `begin
+              insert into testing values (1, 2), (3, 4);
+            end`,
+          },
+        },
+      }],
+    });
+
+    try {
+      await Database.upgrade({schema, runUpgrades: true, readDbUrl: this.dbUrl, writeDbUrl: this.dbUrl});
+      assert.fail('should have failed');
+    } catch (e) {
+      assert(e);
+    }
```

Comment:
`assert.rejects` is useful here -- otherwise, while I think this is right, it's very easy to get this construct wrong.